rust/openvasctl/src/openvas_redis.rs

```diff
@@ -45,6 +45,12 @@ pub trait KbAccess {
         value: T,
     ) -> RedisStorageResult<()>;
     fn kb_id(&self) -> RedisStorageResult<u32>;
+    fn results(&mut self) -> RedisStorageResult<Vec<String>> {
+        Ok(Vec::new())
```

Comment:
what's the reasoning for the default implementation?

Reply:
For the FakeRedis struct.